.travis.yml

```diff
@@ -0,0 +1,65 @@
+language: python
+
+python:
+  - "3.6"
+  - "3.7"
+  - "3.8"
+  - "3.9"
+
+cache:
+  - pip
+
+before_install:
+  - python -m pip install --upgrade pip
+
+install:
+  - pip install -e ."[travis]"
+  - python -c 'import himalaya; print(himalaya.__version__)'
+
+script:
+  # install necessary packages to run tests and get coverage
+  - travis_retry pip install -q -U pytest pytest-cov codecov
+  - pytest --cov=./
+  # install packages for docs and make docs
+  #- travis_retry pip install -q ipython Sphinx sphinx-gallery numpydoc
+    #- cd docs && make html && cd ..
+  # Tell GitHub not to use jekyll to compile the docs
+  # - touch docs/_build/html/.nojekyll
+
+after_success:
+  - codecov
+
+# deploy:
+#   - provider: pages
+#     skip_cleanup: true
+#     github_token: $GITHUB_TOKEN  # Set in travis-ci.org dashboard, marked secure
+#     keep-history: true
+#     on:
+#       branch: main
+#       tags: true
+#       # push only once
+#       python: "3.9"
+#     local_dir: docs/_build/html/
+# 
+#   - provider: pages
+#     skip_cleanup: true
+#     github_token: $GITHUB_TOKEN
+#     keep-history: true
+#     on:
+#       branch: main
+#       tags: true
+#       # push only once
+#       python: "3.9"
+#     local_dir: docs/_build/html/
+#     repo: pycortex/pycortex.github.io
```

Comment:
What is `pycortex` ? ;)

Reply:
ah sorry these are commented for a future change (so that I remember to enable the push to gh-pages). I'll rename it to what it should become ;)